### DISCUSSION TITLE
[Bug] No Icons In App

### DISCUSSION THREAD

**Discussion by hbednar** [evilsocket/opensnitch#998] [General]
**Describe the bug**
None of the icons are appearing with in the app.

I have read through the wiki and looked at [#53](https://github.com/gustavo-iniguez-goya/opensnitch/issues/53) as its the same issue i am having.
But, none of the suggestions have worked for me and the icon theme is set every where i have looked and i have installed qt5-style-plugins, qt5ct and qt-material.

Icon Theme  : Papirus [GTK2/3]

Include the following information:
 - OpenSnitch version: 1.6.0
 - OS: Debian
 - Version: sid
 - Window Manager: i3
 - Kernel version: 6.3.0-2

**Post error logs:** 

	~ OpenSnitch GUI - 1.6.0 ~
	protobuf: 4.21.12 - grpc: 1.56.0
`
Using theme: 3 dark_cyan.xml inverted: False
Using server address: unix:///tmp/osui.sock
gRPC Max Message Length: None
                  Bytes: 4194304
is new file, or IN MEMORY, setting initial schema version
setting schema version to: 2
setting schema version to: 2
schema version: 2
db schema is up to date
Loading translations: /usr/lib/python3/dist-packages/opensnitch/utils/../i18n locale:
using IPASN DB: /usr/lib/python3/dist-packages/data/ipasn_20140513_v12.dat.gz
is new file, or IN MEMORY, setting initial schema version
setting schema version to: 2
setting schema version to: 2
schema version: 2
db schema is up to date
new node connected, listening for client responses... /local
`
**Screenshots**
![2023-07](https://github.com/evilsocket/opensnitch/assets/71215137/0a385731-cdcb-4b65-bfd5-073c2c49cf17)

**Comment by gustavo-iniguez-goya** [comment]
Hi @hbednar ,

Launch the GUI as follow please, and let me know if the icons show up: `~ $ XDG_CURRENT_DESKTOP=GNOME opensnitch-ui`

Another solution: 
 - Install qt5ct (apt install qt5ct) and launch it
 - Go to the "Icon Theme" tab, select an icon theme an click on Apply. 
 
Launch the GUI as follow: `~ $ QT_QPA_PLATFORMTHEME=qt5ct`

If any of these methods work, add the variable to your `.bashrc` or `/etc/environment` file.

Anyway, there's bug that prevents from using Qt's builtin icons; I'll fix it ASAP.

**Comment by hbednar** [comment]
@gustavo-iniguez-goya 
Thanks for your help, i tried your suggestions and `XDG_CURRENT_DESKTOP=GNOME` was the only one that worked. 
It might be worth updating the documentation or adding it to the requirements tool `opensnitchd -check-requirements`.

**Reply by gustavo-iniguez-goya** [reply]
I updated it yesterday :) 
https://github.com/evilsocket/opensnitch/wiki/GUI-known-problems#no-icons-on-the-gui

I think I'll also add a warning, at least if XDG_SESSION_TYPE environment variable is not set.
